OrbitLinuxTracing/TracerThread.h

```diff
@@ -94,10 +102,12 @@ class TracerThread {
   bool trace_context_switches_ = true;
   bool trace_callstacks_ = true;
   bool trace_instrumented_functions_ = true;
+  bool trace_gpu_driver_events_ = true;
```

Comment:
Is it intended that this is now enabled by default because it doesn't log a lot of events anymore?

Reply:
No, this was accidentally left in.